compiler/nilcheck.nim

```diff
@@ -0,0 +1,624 @@
+#
+#
+#           The Nim Compiler
+#        (c) Copyright 2017 Andreas Rumpf
```

Comment:
```suggestion
#        (c) Copyright 2019 Andreas Rumpf
```

Reply:
Hm, maybe , not sure how this is maintained